pkg/oci/client/compute.go

```diff
@@ -106,14 +105,19 @@ func (c *client) GetPrimaryVNICForInstance(ctx context.Context, compartmentID, i
 
 		for _, attachment := range resp.Items {
 			if attachment.LifecycleState != core.VnicAttachmentLifecycleStateAttached {
-				glog.Infof("VNIC attachment %q for instance %q has a state of %q (not %q)",
-					*attachment.Id, instanceID, attachment.LifecycleState, core.VnicAttachmentLifecycleStateAttached)
+				c.logger.With(
```

Comment:
You should create a context logger at the beginning of this method call with instance (and maybe compartment id) as fields.